src/SourceBuild/content/repo-projects/dotnet.proj

```diff
@@ -0,0 +1,52 @@
+<Project>
+  <Import Project="Sdk.props" Sdk="Microsoft.NET.Sdk" />
+
+  <PropertyGroup>
+    <RepoApiImplemented>false</RepoApiImplemented>
```

Comment:
```suggestion
```

Think this can go away with our other PR, right?

Reply:
Yes, that's being removed in my other PR.